TLCGen/Views/Tabs/PrioriteitsTab/DataTypes/PrioIngreepView.xaml

```diff
@@ -200,6 +199,20 @@
                         <TextBlock Text="Overig" Grid.ColumnSpan="2" Margin="5" Padding="0" TextDecorations="Underline" />
                         <Label Content="Check op wagen nummer" Grid.Row="1" />
                         <CheckBox IsChecked="{Binding Path=CheckWagenNummer,UpdateSourceTrigger=PropertyChanged}" Margin="5" />
+                        <Grid Grid.ColumnSpan="2" Visibility="{Binding Path=HasKAROV,Converter={StaticResource Bool2VisConv}}" >
```

Comment:
Merk op dat `AutoGrid` al kolommen aanmaakt via `Columns="Auto,*"`.  Ook dit zal ik vereenvoudigen na de merge

Reply:
Er is een geneste Grid omdat er twee visibility condities zijn: HasKAROV en KARSignaalGroepNummersInParameters. De een werkt nu op de geneste Grid, de ander op Label en TextBox. Wanneer 1 van beide ontbreekt mag de optie niet getoond worden (vind ik tenminste ;-) ).
Ik heb gepoogd om eea te maken met een booleanAndConverter (via MultiValueConverter), maar dat kreeg ik niet werkend.